doc/api/test.md

```diff
@@ -291,6 +291,23 @@ test('a test that creates asynchronous activity', (t) => {
 });
 ```
 
+## Watch mode
+
+<!-- YAML
+added: REPLACEME
+-->
+
```

Comment:
I wonder if we should add an explicit `> Stability: 0 - Experimental` notice, as it is not unlikely that this feature would get out of experimental after the test runner.

Reply:
Yeah sound right 👍